client/app/app.js

```diff
@@ -202,34 +202,125 @@ angular
         $translateProvider.use(defaultLanguage);
         $translateProvider.fallbackLanguage("fr_FR");
     })
-    .config(($transitionsProvider, $httpProvider) => {
-        $httpProvider.interceptors.push("translateInterceptor");
 
+    // ui router modal layout definition
+    .run(($stateRegistry, $urlService) => {
+        /**
+         *  As initial URL synchronization is delayed we can check all modal layout states and check if toChilds attribute is setted. For these states we will need to create new states as follow:
+         *  We will take the direct parent of the modal layout state and create new states with the same layout configuration to all of its child states.
+         */
+        const layoutStates = _.filter($stateRegistry.states, (state) => _.get(state, "layout.toChilds") === true || _.get(state, "layout.toChilds", []).length);
+        const getChildStates = (parentStateName) => _.filter($stateRegistry.states, (state) => _.startsWith(state.name, `${parentStateName}.`));
```

Comment:
Same here, you should keep only the name `({ name }) =>`, it's the only property used in the function :)